spec/shadow/index.html

```diff
@@ -1284,7 +1290,7 @@
               <ol>
                 <li>If <var>ELEMENT</var> is a shadow host:
                   <ol>
-                    <li>Unless its shadow root’s delegatesFocus flag is set, append<var> ELEMENT</var> to <var>MERGED-NAVIGATION-ORDER</var>.</li>
+                    <li>If it is focusable, its <a>tabindex</a> value is not negative, and its <a>shadow root</a>’s delegatesFocus flag is not set, append <var>ELEMENT</var> to <var>MERGED-NAVIGATION-ORDER</var>.</li>
```

Comment:
This condition in the sentence might be confusing. Could you say ELEMENT has to satisfies every conditions explicitly?